extensions/Worker.Extensions.Tables/release_notes.md

```diff
@@ -4,6 +4,8 @@
 - My change description (#PR/#issue)
 -->
 
-### Microsoft.Azure.Functions.Worker.Extensions.Tables <version>
+### Microsoft.Azure.Functions.Worker.Extensions.Tables 1.2.2
```

Comment:
Should this value match the 1.3.0 version you specified below?

Reply:
Yes! Will update.